.drone.star

```diff
@@ -26,7 +26,7 @@ config = {
 				'sqlite',
 				'mariadb:10.2',
 			],
-			'coverage': False
+			'coverage': True
```

Comment:
```suggestion
			'coverage': False
```

We do not usually bother sending the coverage from multiple PHP versions. The coverage file names for 7.3 and 7.4 are not mentioned in sonar-project.properties.